pasportaservo/settings/dev.py

```diff
@@ -15,6 +16,12 @@
     'localhost',
     '127.0.0.1',
 ]
+try:
+    with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
+        s.connect(('10.0.0.10', 0))
+        ALLOWED_HOSTS.append(s.getsockname()[0])
+except (OSError, IndexError):
+    pass
```

Comment:
Kial `socket` utilas?

Reply:
laŭ miaj esploroj kaj permanaj provoj, tio estas la sola maniero akiri en fidinda maniero la lokaretan (LAN) IP-adreson de la maŝino. videbligi la disvolvigan servilon (`manage.py runserver`) en la loka reto mi trovis tre utila ĉar tiamaniere mi povas testi la disvolvigatan version rekte sur realaj aparatoj (ekz., sur poŝtelefonoj) kaj mi tediĝis aldoni tiun ĉi agordon refoje.

Reply:
Mojose :)
Ĉu vi pensas ke utilus havi paragrafon (alineon?) pri tio ekz en la README?

Reply:
farita en c61016a